yt_dlp/extractor/kukululive.py

```diff
@@ -0,0 +1,156 @@
+from urllib.parse import parse_qs
```

Comment:
I personally don't like `from` imports from stdlib. But it's become widespread in code now, so I won't say never to use it. But especially for `parse_qs`, there is a `utils.parse_qs` which is easily confused with this

Reply:
Agreed. I don't like having to check imports to know which function is being used; same goes for `urljoin` etc